src/D2Reader/Readers/UnitReader.cs

```diff
@@ -139,9 +139,9 @@ public D2MonsterData GetMonsterData(D2Unit unit)
             return reader.Read<D2MonsterData>(unit.UnitData);
         }
 
-        private D2ItemDescription GetItemDescription(Item item)
+        public D2ItemDescription GetItemDescription(D2Unit item)
```

Comment:
Fine for now =)

But I d prefer if this could stay Item (more clear that 100% item comes in, and never another unit), but the change in InventoryReader makes this necessary atm.